src/components/Firealerts/Form/FirealertsForm.tsx

```diff
@@ -0,0 +1,142 @@
+        key = key.slice(1);
+        const newAlertData = { ...alertData };
+        updateFromStringPath(newAlertData, key, value);
+        setAlertData(newAlertData);
+    }
+
+    
+    const sendAlert = async () => {
+        const event = generateCloudEventWithData(selectedAlert, alertData);
+        const payload = { events: [event] }
+        const url = `//${config.hostAndPort}/google/publishEvents`
+        console.log("Posting event to: " + url);
+        const res = await fetch(url, {
+            method: "POST",
+            body: JSON.stringify(payload)
+        });
+        console.log(res); // TODO(gburroughs): Add notification of success/error
+    }
+
+    const getTriggerName = (trigger: FirealertsTrigger): string => {
+        const tokens = trigger.triggerName.split("-");
+        return tokens[tokens.length - 2];
+    }
+
+    if (!triggers || triggers.length === 0) {
+        return (
+            <GridCell span={12}>
+                <ZeroState />
+            </GridCell>
+        );
+    }
+    return (
+        <>
+            <GridCell span={12}>
+                {!triggers || triggers.length === 0 ? <ZeroState /> :
+                    <GridRow>
+                        <GridCell span={12}>
+                            <Typography use="headline6">Send Test {alertConfiguration[selectedAlert].name} Alert</Typography>
+                        </GridCell>
+                        <GridCell span={12}>
+                            <Callout>For more information on what the fields mean see <a href={alertConfiguration[selectedAlert].link}>here</a></Callout>
+                        </GridCell>
+                        <GridCell span={12}>
+                            <Card className="container">
+                                <Select outlined label="Select Alert Type" value={selectedAlert} options={alertsList} onChange={(e: ChangeEvent<HTMLSelectElement>) => setSelectedAlert(e.target.value as FirealertsType)}></Select>
+                                <Typography use="body2" theme="textSecondaryOnBackground"> Triggers: {implementedAlerts[selectedAlert]?.map(getTriggerName).reduce((p, c) => p + ", " + c)}</Typography>
+                                <form className="eventForm">
+                                    {createJSONForm(alertData, 1, "", updateData)}
+                                </form>
+                                <GridRow>
+                                    <GridCell span={2}>
+                                        <Button label="Send Alert" unelevated icon="send" onClick={sendAlert} />
+                                    </GridCell>
+                                </GridRow>
+                                <FirealertsLog />
+                            </Card>
+                        </GridCell>
+                    </GridRow>
+                }
+            </GridCell>
+        </>
+    );
+}
+
+const updateFromStringPath = (obj: any, path: string, value: any) => {
```

Comment:
nit: Use `lodash.set` for this

Reply:
Didn't know this existed! Thanks for pointing it out.